v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FExtractionsAndTransforms.scala

```diff
@@ -48,9 +46,16 @@ object V2FExtractionsAndTransforms {
   ): SCollection[(String, Msg)] = {
     msgAndFilePaths.map {
       case (path, msg) =>
+        // change to snake case
+        val withSnakeCase = MsgTransformations.keysToSnakeCase(msg)
+        // rename fields
+        val withRenamedFields =
+          MsgTransformations.renameFields(Map("var_id" -> "id"))(withSnakeCase)
         // extract variant fields
         val withExtractedFields =
-          MsgTransformations.extractFields(v2fConstant.variantFieldsToExtract)(msg)
+          MsgTransformations.extractFields(v2fConstant.variantFieldsToExtract)(
+            withRenamedFields
+          )
         // convert to longs
         val withLongs =
```

Comment:
Are these type conversions redundant at the point where we call `extractAndTransformVariants`? If not, could we refactor to make them redundant?

Reply:
They're not redundant at this point, and yeah I can pull them earlier for DRYer code.